In this file, I want to add the following variables:
- z_min_safety : minimum z value for the fork to be safe
- or the point cloud of the container : to add the global safety. 